iothub/service/src/Twin/DeviceCapabilities.cs

```diff
@@ -6,12 +6,12 @@
 namespace Microsoft.Azure.Devices
 {
     /// <summary>
-    /// Status of Capabilities enabled on the device
+    /// Status of capabilities enabled on the device.
     /// </summary>
     public class DeviceCapabilities
     {
         /// <summary>
-        /// IoT Edge capability.
+        /// IoT edge capability.
```

Comment:
Sorry, Edge should be capitalized in this case, as it is a product.

When in doubt, search for one of our [docs pages](https://docs.microsoft.com/en-us/azure/iot-edge/about-iot-edge?view=iotedge-2020-11) to see how it is cased. In this case, they are all capitalized except one case where not referring to the product, yet the concept.